src/assignments/benson/lesson_2_answers.clj

```diff
@@ -56,4 +68,37 @@
        ;; and puts the entire sequence into a set
        (vals)
        (filter #(>= (count %) 2))
-       (into #{})))
\ No newline at end of file
+       (into #{})))
+
+
+;; Returns all consecutive subsequences as nested vectors
+(defn get-consecutive-subseqs
+  [seqs]
+  (reduce
+   ;; Function that groups subsequences as nested vectors together in a vector
+   (fn [seqs number]
+     ;; If the current number is larger than the previous number by one (in consecutive sequence)
+     ;; Then conjoin it to the last nested vector
+     ;; Else associate a new nested vector containing the number at the end
+     (if (= number (inc (last (last seqs))))
```

Comment:
When you see a bunch of (last seqs), (rest seqs) and (first seqs) in your code, it often indicates that you should be destructuring somewhere.  Can you think how this might be improved?

Your code is very well laid out, and the names of functions and vars are well chosen.  I can follow your intent.  Well done!